lib/models/session.js

```diff
@@ -40,6 +40,10 @@ module.exports = function getSession(provider) {
       return this.loginTs;
     }
 
+    meta() {
```

Comment:
this is not necessary, there's `metaFor`

Reply:
Removed